### PR TITLE
AFKR-15: Replaced Frais d’inscription by Frais de consultation

### PR DESCRIPTION
**Pull request by rubailly**
Implementing changes requested by Serge:

- We proposed to replace “Frais d’inscription” by “Frais de consultation” in the two places where it written.

### CODE REVIEW

File: configuration/openmrs/i18n/registration/.~lock.locale_en.json# [hunk 1]
```diff
@@ -0,0 +1 @@
+,rbailly,rbailly-ThinkPad-E460,02.07.2020 12:19,file:///home/rbailly/.config/libreoffice/4;
```
Comment (rbuisson) [inline configuration/openmrs/i18n/registration/.~lock.locale_en.json#:1]: to remove.

File: configuration/openmrs/i18n/registration/locale_fr.json [hunk 2]
```diff
@@ -0,0 +1,131 @@
...
+  "ATTRIBUTE_SEARCH": "Attributs",
+  "CHOOSE_ANSWER_FROM_DROPDOWN_LABEL": "Selectionner une réponse",
+  "CLICK_PATIENT_PHOTO_LABEL": "Prendre photo",
+  "CONCEPT_NOT_FOUND_MESSAGE": "Impossible de trouver le Concept",
+  "CONFIRM_PATIENT_PHOTO_LABEL": "Valider",
+  "DELETE_LABEL": "Delete",
+  "CUSTOM_SEARCH": "Entrer",
+  "DISPLAY_CONTROL_ERROR_MESSAGE_KEY": "Pas {{ displayControl }} pour ce patient.",
+  "EDIT_LABEL": "Modifier",
+  "FEE_INFORMATION_LOCALE_KEY": "Frais de consultation",
```
Comment (rbuisson) [inline configuration/openmrs/i18n/registration/locale_fr.json:13]: I think we should rather change that in the parent Haiti config, what do you think?

**Review by rbuisson — CHANGES_REQUESTED** [review]
Should be done at the Haiti level.

File: configuration/openmrs/i18n/registration/locale_fr.json (hunk 2 above)
Comment (rubailly) [reply configuration/openmrs/i18n/registration/locale_fr.json:13]: That would be great actually. Cause I had to duplicate that registration directory in i18n folder into easyDoc to override what is in the parent.

File: configuration/openmrs/i18n/registration/.~lock.locale_en.json# (hunk 1 above)
Comment (rubailly) [reply configuration/openmrs/i18n/registration/.~lock.locale_en.json#:1]: Yup. Thanks.

**Comment by rubailly** [comment]
I moved this PR to the Haiti repo

**Pull request closed without merging**